src/others/helpers/decode-aid-request.ts

```diff
@@ -71,7 +84,9 @@ export const decodeAidRequestGroupedByCategoryWithTotal = (dictionary: Dictionar
 
 const decodeLocation = (locations: Location[], cityId: ID): DecodedLocation => {
   const location = locations.find((location) => String(location.id) === String(cityId));
-  return omit(location, "id");
+  if (!location) throw new Error(`Locatin: ${location} is not defined`);
+
+  return location;
 };
 
 const decodeCategory = (categories: Supply[], category_id: ID): string => {
```

Comment:
why do we still have this function that does a linear scan? why not switch the users to use DictionaryContext?

Reply:
Explanation above ☝️